shogun-lib/src/main/java/de/terrestris/shoguncore/config/JacksonConfig.java

```diff
@@ -1,19 +1,67 @@
 package de.terrestris.shoguncore.config;
 
 import com.bedatadriven.jackson.datatype.jts.JtsModule;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.vladmihalcea.hibernate.type.util.ObjectMapperSupplier;
+import de.terrestris.shoguncore.annotation.JsonSuperType;
 import org.locationtech.jts.geom.GeometryFactory;
 import org.locationtech.jts.geom.PrecisionModel;
+import org.reflections.Reflections;
+import org.reflections.scanners.SubTypesScanner;
+import org.reflections.scanners.TypeAnnotationsScanner;
+import org.reflections.util.ClasspathHelper;
+import org.reflections.util.ConfigurationBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.annotation.PostConstruct;
+import java.util.Set;
+
 @Configuration
-public class JacksonConfig {
+public class JacksonConfig implements ObjectMapperSupplier {
+
+    private static ObjectMapper mapper;
 
     @Bean
-    public JtsModule jtsModule() {
+    public ObjectMapper objectMapper() {
+        init();
+        return mapper;
+    }
+
+    @Bean
+    public static JtsModule jtsModule() {
         GeometryFactory geomFactory = new GeometryFactory(new PrecisionModel(10), 4326);
 
         return new JtsModule(geomFactory);
     }
 
+    @Override
+    public ObjectMapper get() {
+        return objectMapper();
+    }
+
+    @PostConstruct
+    public static void init() {
+        if (mapper == null) {
+            mapper = new ObjectMapper().registerModule(jtsModule());
+
+            mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
According to the [docs](https://docs.spring.io/spring-boot/docs/2.4.2/reference/html/howto.html#howto-customize-the-jackson-objectmapper) the default mapper created by spring has also set/disabled the following features:

* `MapperFeature.DEFAULT_VIEW_INCLUSION`
* `DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES`
* `SerializationFeature.WRITE_DATES_AS_TIMESTAMPS`

If they aren't disabled by default anyway, I would suggest to also disable them.